multiqc/multiqc.py

```diff
@@ -176,6 +176,14 @@
 @click.option("-q", "--quiet", is_flag=True, help="Only show log warnings")
 @click.option("--profile-runtime", is_flag=True, help="Add analysis of how long MultiQC takes to run to the report")
 @click.option("--no-ansi", is_flag=True, help="Disable coloured log output")
+@click.option(
+    "--custom-css-files",
```

Comment:
I don't think that you need this twice..

Reply:
You're probably right. Above:
```
@click.option(
    "--cl-config", "--cl_config", type=str, multiple=True, help="Specify MultiQC config YAML on the command line"
)
```

But lemme try taking it out :)

Reply:
Yeah, past-me probably didn't know better. Only needed when doing short form + long form if I remember correctly.